src/components/_top-container/version-history/selector/selected-datetime/VersionSelectorDateTime.tsx

```diff
@@ -46,13 +43,16 @@ export const VersionSelectorDateTime = ({
     return (
         <>
             <div className={style.left}>
-                <Label>{'Velg tid og dato:'}</Label>
+                <Label id={'version-selector-label'}>
+                    {'Velg tid og dato:'}
+                </Label>
                 <input
                     type={'time'}
                     className={style.time}
                     onChange={(e) => {
                         setTimeSelected(e.target.value);
                     }}
+                    aria-labelledby={'version-selector-label'}
                     value={timeSelected.slice(0, 5)}
                 />
```

Comment:
Er `id` knyttet til logikk her? 

Om ikke, kan kanskje Label og input kobles sammen med `for` på `<label>` og `id` på `<input>`

Reply:
Denne label'en gjelder for to input-felter, så tror ikke det vil funke.